db/large_data_handler.cc

```diff
@@ -122,8 +130,8 @@ static future<> try_record(std::string_view large_table, const sstables::sstable
         extra_fields_str += format(", {}", field);
         extra_values += ", ?";
     }
-    const sstring req = format("INSERT INTO system.large_{}s (keyspace_name, table_name, sstable_name, {}_size, partition_key, compaction_time{}) VALUES (?, ?, ?, ?, ?, ?{}) USING TTL 2592000",
-            large_table, large_table, extra_fields_str, extra_values);
+    const sstring req = format("INSERT INTO system.large_{}s (keyspace_name, table_name, sstable_name, {}, partition_key, compaction_time{}) VALUES (?, ?, ?, ?, ?, ?{}) USING TTL 2592000",
+            large_table, column, extra_fields_str, extra_values);
```

Comment:
Is passing the column name still required now that rows_count is kept in the large partition row?